packages/k8s.contrib.crd/generate.pkl

```diff
@@ -148,11 +150,17 @@ fixed modules: Listing<ModuleGenerator> = new {
   }
 }
 
+fixed schema: SchemaGenerator = new {
+  crds = _crds
+  baseApiGroup = module.baseApiGroup
+}
+
 output {
   text = throw("The JSON Schema generator only works with multiple-file output. Try running again with the -m option.")
   files {
+    ["k8sSchema.pkl"] = schema.moduleNode.output
```

Comment:
I think it'd be good to put this under a flag; many users likely would not care for this.